CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## [1.9.4] - 2019-07-30
+### Fix
+- Fix 'busbar logs' issue with dashes on application name
```

Comment:
Note that `busbar show` isn't written in quotes on line 9 below. Also see tiny fix below.

```suggestion
- Fix busbar logs issue with dashes in the application name
```